app/services/taxonomy/publish_taxon.rb

```diff
@@ -25,6 +25,14 @@ def publish
       raise(InvalidTaxonError, I18n.t('errors.invalid_taxon'))
     end
 
+    def unpublish(type:, alternative_path: nil)
```

Comment:
I think a separate class for every action is too much boilerplate. I'd rather rename this one to something that is appropriate for any any publishing api actions on a taxon. But I'm having trouble with words today...

TaxonPublishingApiRequestMakerThing?? 🆘 😞